src/runtime_src/core/common/CMakeLists.txt

```diff
@@ -94,8 +94,14 @@ install(TARGETS xrt_coreutil
   RUNTIME DESTINATION ${XRT_INSTALL_BIN_DIR}
 )
 
-install(TARGETS xrt_coreutil xrt_coreutil_static
+install(TARGETS xrt_coreutil
```

Comment:
It seems a little weird to me that when `XRT_DEV_COMPONENT` is set to `xrt-dev`, you set the `xrt-dev` package to depend on `xrt` (the runtime package), but you also include all of the runtime package targets in what is being installed into `xrt-dev`.  It seems like you would have all the runtime files inside the `xrt-dev` package and not need the depenence on the runtime package.

If you were going to make the `xrt-dev` package depend on the `xrt` runtime package, I would think `xrt-dev` would only include the:
* headerfiles
* static libraries (unless you're splitting them out into `xrt-static`)
* cmake and pkg-config definition files